.github/workflows/compatibility.yml

```diff
@@ -31,13 +31,22 @@ jobs:
           source venv/bin/activate
           python -m pip install --upgrade pip
           pip install hatch pyqt5
-          pip install git+https://github.com/PyMoDAQ/PyMoDAQ.git@${{ matrix.pymodaq-version }}
-
+          if [[ "${{ matrix.pymodaq-version }}" = "latest" ]]; then
+            pip install git+https://github.com/PyMoDAQ/PyMoDAQ.git
+          else
+            pip install git+https://github.com/PyMoDAQ/PyMoDAQ.git@${{ matrix.pymodaq-version }}
+          fi
+          pip install -e .
+          
       - name: Run script
         run: |
           source venv/bin/activate
-          python src/pymodaq_plugin_manager/compatibility_checker.py "git+https://github.com/PyMoDAQ/PyMoDAQ.git@${{ matrix.pymodaq-version }}"
-
+          if [[ "${{ matrix.pymodaq-version }}" = "latest" ]]; then
+            pymodaq="git+https://github.com/PyMoDAQ/PyMoDAQ.git"
+          else
+            pymodaq="git+https://github.com/PyMoDAQ/PyMoDAQ.git@${{ matrix.pymodaq-version }}"
+          fi
+            python src/pymodaq_plugin_manager/compatibility_checker.py "${pymodaq}"
```

Comment:
because it is installed one could use : `python -m pymodaq_plugin_manager.compatibility`, no?